homeassistant/components/switch/rfxtrx.py

```diff
@@ -0,0 +1,126 @@
+"""
+homeassistant.components.switch.rfxtrx
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Support for Rfxtrx switch.
+
+Configuration:
+
+To use Rfxtrx switchs you will need to add the following to your
+configuration.yaml file.
+
+switch:
+  platform: rfxtrx
+
+  devices:
+    ac09c4f1: Bedroom Door
+    ac09c4f2: Kitchen Door
+    ac09c4f3: Bathroom Door
+
+*Optional*
+
+  # Automatic add new switch
+  automatic_add: True
+
+"""
+import logging
+import homeassistant.components.rfxtrx as rfxtrx
+from RFXtrx import LightingDevice
+
+from homeassistant.components.switch import SwitchDevice
+from homeassistant.util import slugify
+
+REQUIREMENTS = ['https://github.com/Danielhiversen/pyRFXtrx/archive/' +
```

Comment:
We can replace the `REQUIREMENTS` in the platforms and just add `DEPENDENCIES = ['rfxtrx']`. This will cause the `rfxtrx` component to be loaded and installed.